docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -97,10 +97,35 @@ async function postFeedback(data) {
 async function postFeedbackOnSlack(data) {
   const { rating, comment, commentedSection } = data;
 
+  const sentData = {
+    callback_id: 'send_feedback',
+    rating,
+    comment,
+    currentLocationURL: window.location.href,
+    commmentSectionURL: `${window.location.origin}${window.location.pathname}#${commentedSection.hash}`,
+    commmentSectionTitle: commentedSection.text,
+  };
   if (!comment || comment.length < 10) {
     return 'ignored';
   }
 
+  try {
+    // await fetch(`${window.location.origin}/.netlify/functions/feedback-management/`, {
+    await fetch(
+      `https://deploy-preview-36472--material-ui.netlify.app//.netlify/functions/feedback-management/`,
+      {
+        method: 'POST',
+        headers: { 'content-type': 'application/x-www-form-urlencoded' },
+        // Seems tricky but it's to match how slack send data
+        body: `payload=${encodeURIComponent(JSON.stringify(sentData))}`,
+      },
+    );
+    return 'sent';
```

Comment:
The `fetch` response could have returned with a non-200 status code. I suppose this should be considered an error as well?

I usually do something like

```tsx
const res = await fetch(url);
if (!res.ok) {
  throw new Error(`HTTP ${res.status}: ${res.statusText}`)
}
```